scripts/env/check-node

```diff
@@ -2,6 +2,15 @@
 
 # Check whether the current node version matches the .nvmrc version, and offer some help if not.
 
+# Check if we should skip this check
+# Required to prevent compressed-size-action@v2 from failing when on a branch that changes node versions
+# The action checks out main to compare against the current branch but does not reset the node version
+# check-node still runs as part of other CI jobs
+if [[ "$SKIP_CHECK_NODE" == "TRUE" ]]; then
+	echo "Skipping check-node"
+	exit 0
+fi
+
```

Comment:
Is there a option to [update `compressed-size-action` upstream](https://github.com/preactjs/compressed-size-action/blob/master/action.yml)?